snap/snapcraft.yaml

```diff
@@ -38,6 +38,7 @@ parts:
     - -DCMAKE_C_FLAGS_RELEASE='-O3 -DNDEBUG -Wa,-mrelax-relocations=no'
     - -DLLVM_ROOT_DIR=../../llvm/install
     - -DLDC_INSTALL_LTOPLUGIN=ON
+    - -DLDC_WITH_LLD=OFF
```

Comment:
here you also want to add `-DLDC_INSTALL_LLVM_RUNTIME_LIBS=ON` I think, such that ASan, libFuzzer, etc, is included as well (for future LDC versions).

Reply:
Is that worth adding now?  I'm not doing anything to build them at this stage (this is planned for some later updates).

Reply:
It's fine to delay it.